docs/userGuide/syntax/code.mbdf

```diff
@@ -125,6 +125,25 @@ To add a heading, add the attribute `heading` with the heading text as the value
 </span>
 </include>
 
+Headings support inline MarkDown some exceptions (i.e. inline code, dim text style, block quotes, headers, lists, paragraphs, footnotes and embeds).
```

Comment:
`with some exceptions`
Could only mention the inline exclusions ( inline code / dim )